<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificarProductoDialog</class>
 <widget class="QDialog" name="ModificarProductoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar Producto</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>        Nombre:   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNombreP"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>        Marca:     </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditMarcaP"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>       Precio:       </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPrecioP"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>       Cantidad:   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCantidadP"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
